### ISSUE TITLE
CircuitBreaker configuration

### ISSUE THREAD

**Issue by ArturChe** [akkadotnet/Akka.Persistence.Redis#87]
From time to time we can observe errors while having problems to connect the Redis:
`Recovery failed due to: Akka.Pattern.OpenCircuitException: Circuit Breaker is open; calls are failing fast    at Akka.Pattern.Open.<Invoke>d__2`1.MoveNext() on message ().`

I have found that there are settings that used if nothing defined in the plugin config
```
 journal-plugin-fallback {
...
 circuit-breaker {
  max-failures = 10
  call-timeout = 10s
  reset-timeout = 30s
}
```
Is it possible to configure a `circuit-breaker` in
```
journal {
 plugin = "akka.persistence.journal.redis"
 redis {
  class = "Akka.Persistence.Redis.Journal.RedisJournal, Akka.Persistence.Redis"
  key-prefix = "akka:shmaka:persistence:"
  configuration-string = "localhost"
}
```
section?

**Comment by Aaronontheweb** [comment]
@Arkatufus we just addressed this in v1.4.13, did we not?

**Comment by Aaronontheweb** [comment]
@ArturChe yes, you should be able to configure these values in the `akka.persistence.journal.redis` setting.
